src/Hangfire.PostgreSql/PostgreSqlStorage.cs

```diff
@@ -123,13 +126,19 @@ public override IEnumerable<IServerComponent> GetComponents()
     {
       yield return new ExpirationManager(this);
       yield return new CountersAggregator(this, Options.CountersAggregateInterval);
+      if (Options.SlidingInvisibilityTimeout.HasValue)
+      {
+        // This is only used to update the sliding invisibility timeouts, so if not enabled then do not use it
+        yield return HeartbeatProcess;
```

Comment:
If it's supposed to run only if enabled, maybe we don't need to initialize it in the constructor in the first place?

Reply:
Updated so only creates the ```Heartbeat``` process if ```UseSlidingInvisibilityTimeout``` is set to true.  

This pretty much broke all the PostgreSqlJobQueueFacts tests because I assume of the shared storage fixture.  So had to set UseSlidingInvisibilityTimeout to true for the entire all those tests and only turn it on/off for the specific tests in question.